hook!-full-game/src/UI/debug/DebugPanel.gd

```diff
@@ -0,0 +1,38 @@
+extends Control
+
+onready var reference_node: = get_node(reference_node_path)
+onready var container: VBoxContainer = $MarginContainer/VBoxContainer
+
+export (NodePath) var reference_node_path: NodePath = "."
+export (PoolStringArray) var properties = []
+
+func _ready() -> void:
+	setup_properties()
+
+
+func _process(delta) -> void:
+	for property_string in properties:
+		update_property(property_string, reference_node)
+
+
+func setup_properties() -> void:
+	clear_property_labels()
+	for property_string in properties:
+		add_property_label(property_string)
+
+
+func clear_property_labels() -> void:
+	for property_label in container.get_children():
+		property_label.queue_free()
+
+func update_property(property: String, reference_node: Node) -> void:
+	var property_label: Label = container.get_node(property.capitalize()) as Label
+	property_label.text = "%s %s: %s"%[reference_node.name, property.capitalize(),
```

Comment:
As the panel only has one reference node, how about having a separate title/node name label at the top of the panel to show the node's name? That way you don't have to repeat the node's name on every line